config/types.go

```diff
@@ -221,3 +221,10 @@ type RawService map[string]interface{}
 
 // RawServiceMap is a collection of RawServices
 type RawServiceMap map[string]RawService
+
+type ParseOptions struct {
```

Comment:
I think it makes a bit more sense to me to keep it in `config`. These options should only be relevant to the parsing side, whereas everything in `project/options` seems to just apply to just project actions.